lib/membrane/aac/parser/helper.ex

```diff
@@ -89,15 +89,59 @@ defmodule Membrane.AAC.Parser.Helper do
       <<_adts::binary-size(adts_size), frame::binary-size(frame_size), rest::binary>> ->
         {:frame, frame, rest}
 
-      _ ->
+      _other ->
         :no_frame
     end
   end
 
-  defp next_timestamp(timestamp, caps) do
+  @spec next_timestamp(any(), AAC.t()) :: AAC.Parser.timestamp_t()
+  def next_timestamp(timestamp, caps) do
     use Ratio
 
     timestamp +
       Ratio.new(caps.samples_per_frame * caps.frames_per_buffer * Time.second(), caps.sample_rate)
   end
+
+  @spec to_adts(AAC.t(), binary()) :: binary()
+  def to_adts(%AAC{} = caps, payload) do
```

Comment:
Maybe `payload_to_adts` or `add_adts_header` or something else that would be a little more descriptive.

Also, I would swithc `caps`  and `payload` so we would have `to_adts(payload, caps)` to be consistent with `next_timestamp`